### ISSUE TITLE
Warning existing folder does not exist

### ISSUE THREAD

**Issue by pepe59** [nimroddolev/chime_tts#229]
## System Information

version | core-2024.12.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.13.0
os_name | Linux
os_version | 6.6.51-haos-raspi
arch | aarch64
timezone | Europe/Prague
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
HACS Data | ok
GitHub API Calls Remaining | 5000
Installed Version | 2.0.1
Stage | running
Available Repositories | 1486
Downloaded Repositories | 46

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 3. května 2025 v 02:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | true
cloud_ice_servers_enabled | true
remote_server | eu-central-1-5.ui.nabu.casa
certificate_status | ready
instance_id | 3e42f135b8e94cceac25531ec6bd3d97
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 14.0
-- | --
update_channel | stable
supervisor_version | supervisor-2024.11.4
agent_version | 1.6.0
docker_version | 27.2.0
disk_total | 109.3 GB
disk_used | 17.8 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | 
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons | Samba share (12.3.2), File editor (5.8.0), Terminal & SSH (9.16.0), Home Assistant Google Drive Backup (0.112.1), Mosquitto broker (6.4.1), Samba Backup (5.2.0), openWakeWord (1.10.0), Piper (1.5.2), Uptime Kuma (0.12.4), ESPHome Device Compiler (2024.11.3), Zigbee2MQTT (1.42.0-2)

</details>

<details><summary>Dashboards</summary>

dashboards | 4
-- | --
resources | 30
views | 13
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 9. prosince 2024 v 18:40
-- | --
current_recorder_run | 15. prosince 2024 v 10:13
estimated_db_size | 106.05 MiB
database_engine | sqlite
database_version | 3.45.3

</details>

<details><summary>Sonoff</summary>

version | 3.8.1 (ffa7e22)
-- | --
cloud_online | 11 / 14
local_online | 9 / 9

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

I don't know if I missed something, but after updating to v1.2.0 I get a warning while playing. Everything works fine.
![nastaveni chime tony](https://github.com/user-attachments/assets/71fe164c-06e7-4715-a41d-0cf485a12267)
folder exists
![slozka tonu](https://github.com/user-attachments/assets/8ed63b73-6908-4028-9772-737f51855820)

### Reproduction steps

1.
2.
3.
...
update to v1.2.0

### Debug logs

```text
Tato chyba pochází z vlastní integrace.

Logger: custom_components.chime_tts.helpers.filesystem
Zdroj: custom_components/chime_tts/helpers/filesystem.py:54
integrace: Chime TTS (dokumentace, problémy)
První výskyt: 11:00:03 (1 výskyty)
Naposledy logováno: 11:00:03

Error: The directory '/www/tony' was not found.
```

**Comment by nimroddolev** [comment]
Hi @pepe59,

This error relates to looking up the folder on your filesystem. I don't quite understand the conext of this error, and it might be nothing given that it doesn't seem to effect the process - but could you please [enable debug logging](https://nimroddolev.github.io/chime_tts/docs/support#debug-logging) and provide me with the debug logs?

**Comment by pepe59** [comment]
Here is the error log. I hope I downloaded it correctly.
Nothing has changed in my HA since the last beta, I just updated Chime
[chime_tts.txt](https://github.com/user-attachments/files/18148590/chime_tts.txt)

**Comment by nimroddolev** [comment]
Thank you. What was the YAML that you used when you called `chime_tts.say`?

**Comment by pepe59** [comment]
I only use the action in automation nothing else
```
target:
  entity_id: media_player.picoretts
data:
  chime_path: /config/www/tony/zvonek.mp3
  offset: 450
  tts_speed: 107
  volume_level: 0.5
  message: |-
    Aktuální čas je {{states('sensor.time')}}. Pokojová teplota {{
    states('sensor.teplota_byt') | replace('.',',') }} stupňů. Venku {{
    states('sensor.venkovni_teplota') | round(0) | replace('-','mínus') }}
  tts_platform: cloud
  cache: true
action: chime_tts.say
```

**Comment by nimroddolev** [comment]
I have released a new beta version, v1.2.2-beta1 with a fix for this issue.

**Comment by pepe59** [comment]
Yes I can confirm that the error no longer appears.
Thank you very much
